core/domain/skill_domain.py

```diff
@@ -653,3 +653,48 @@ def to_dict(self):
             'skill_model_created_on': self.skill_model_created_on,
             'skill_model_last_updated': self.skill_model_last_updated
         }
+
+
+class SkillsMastery(object):
+    """Domain object for an Oppia Skills Mastery."""
+
+    @classmethod
+    def get_model_id(cls, user_id, skill_id):
```

Comment:
This should be part of the model class. It's an internal detail that should not be exposed beyond the storage layer. See other models that have composite IDs for a pattern on how to do this.

Reply:
ok